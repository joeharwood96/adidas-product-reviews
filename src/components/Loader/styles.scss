.loader {
    text-align: center;
    position: absolute;
    top: 50%;
    left: 50%;

    &-logo {
        height: 4rem;
        margin-bottom: 4rem;
    }
}