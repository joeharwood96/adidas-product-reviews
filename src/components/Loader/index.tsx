import React from 'react'
import { TailSpin } from  'react-loader-spinner'
import logo from '../../assets/Adidas_Logo.svg'
import "./styles.scss"

export default function Loader() {

    return (
        <div className="loader">
            <div className="loader-center">
                <img src={logo} alt="Adidas logo" className="loader-logo" />
                <TailSpin
                    height="100"
                    width="100"
                    color='black'
                    ariaLabel='loading'
                />
            </div>
        </div>
    )
}