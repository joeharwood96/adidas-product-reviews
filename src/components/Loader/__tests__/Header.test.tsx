import React from "react";
import Loader from "../index";

describe("<Loader />", () => {
    test("should display the loader", () => {
        const component = <Loader />
        expect(component).toMatchSnapshot()
    });
});