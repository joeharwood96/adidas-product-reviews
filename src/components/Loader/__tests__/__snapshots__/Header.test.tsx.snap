// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should display the loader 1`] = `<Loader />`;
